entity-types/ext-temporal_cloud/definition.yml

```diff
@@ -0,0 +1,21 @@
+---
+domain: EXT
+type: TEMPORAL_CLOUD
```

Comment:
Don't know the product.
Is there any public documentation available ?

Wondered if 'temporal cloud' is a reasonable entity type or if there are other primitives that make up such a cloud ?

Reply:
Hi @andre-nr ,
Here is the temporal cloud public [documentation](https://docs.temporal.io/cloud).

tagging @seemantk